test/integration/image-component/default/test/index.test.js

```diff
@@ -1013,12 +1013,23 @@ function runTests(mode) {
   it('should load the image when the lazyRoot prop is used', async () => {
     let browser
     try {
-      //trying on '/lazy-noref' it fails
       browser = await webdriver(appPort, '/lazy-withref')
 
       await check(async () => {
         const result = await browser.eval(
-          `document.getElementById('myImage').naturalWidth`
+          `document.getElementById('myImage2').naturalWidth`
```

Comment:
Lets check `myImage1` too?

Reply:
Good Morning! "myImage1" is not supposed to be loaded...

Reply:
I see. Lets add a test to ensure it is never loaded 🙂

Reply:
Done